package-overrides/github/angular-ui/ui-select@0.14.2.json

```diff
@@ -0,0 +1,19 @@
+{
+  "dependencies": {
+    "css": "*",
+    "angular": "^1.3"
+  },
+  "directories": {
+    "lib": "dist"
```

Comment:
It's best to avoid adding this if you can as in case there are users using this already it will break for them.